doc/example_architecture/parameter_component/component-parameter_component-implementation.adb

```diff
@@ -57,4 +57,21 @@ package body Component.Parameter_Component.Implementation is
       Put_Line ("Oh, no! A parameter was sent that was malformed.");
    end Invalid_Parameter;
 
+   -- This function is called when the parameter operation type is "Validate". The default implementation of this
+   -- subprogram in the implementation package is a function that returns "Valid". However, this function can, and should be
+   -- overridden if something special needs to happen to further validate a parameter. Examples of this might be validation of
+   -- certain parameters beyond individual type ranges, or performing other special functionality that only needs to be
+   -- performed after parameters have been validated.
+   overriding function Validate_Parameters_Action (
+      Self : in out Instance;
+      Start_Count : in Packed_U16.U;
+      Hello_World_Value : in Packed_U16.U
+   ) return Parameter_Validation_Status.E is
+      Ignore : Instance renames Self;
```

Comment:
You probably also need to ignore the other values here too, no? You can do this by naming them `Ignore_Start_Count`, etc.

Reply:
Addressed in [2008566](https://github.com/lasp/adamant/pull/64/commits/2008566276f5ade50bfb38842dcb8107d3c220ff). Implementation moved to `.ads`.